posthog/queries/session_recordings/session_recording_list_from_replay_summary.py

```diff
@@ -135,6 +135,8 @@ def build_event_filters(self) -> SummaryEventFiltersSQL:
             )
             joining = "OR" if index > 0 else ""
             condition_sql += f"{joining} ({this_entity_condition_sql})"
```

Comment:
Are the brackets necessary here then? It feels like we should be moving them rather than adding more? 
```suggestion
            condition_sql += f"{joining} {this_entity_condition_sql}"
```

Reply:
Funnily enough CH adds those brackets back in 

```
explain syntax
    select *
    from events
    where
        team_id = 1 and (event = '$pageview' or event = '$pageleave')
    limit 1
```
gives

<img width="669" alt="Screenshot 2023-07-25 at 10 49 19" src="https://github.com/PostHog/posthog/assets/984817/b7dfb9d6-29db-49bd-ab1a-fa1270a7506d">

But we're at least not shuffling those bytes to CH for every query and it's probably more readable for humans when they come to it so 👍